app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,38 @@
+class CommentsController < ApplicationController
+
+
+  def post_comments
```

Comment:
Please see my comments about writing non-CRUD actions as public methods in your resource controllers.